translations/nl-be.json

```diff
@@ -519,5 +521,11 @@
   "explanation": "Toelichting",
   "error-delete-agenda": "Er ging iets mis tijdens het verwijderen van de agenda. Contacteer de administrator of probeer het opnieuw.",
   "no-themes-selected": "Geen thema's geselecteerd",
-  "no-themes-selected-message": "Er zijn nog geen thema's geselecteerd. Bent u zeker dat u het kort bestek wilt opslaan?"
+  "no-themes-selected-message": "Er zijn nog geen thema's geselecteerd. Bent u zeker dat u het kort bestek wilt opslaan?",
+  "monitoring-id": "ID",
+  "monitoring-type": "Type",
+  "monitoring-service": "Service",
+  "monitoring-timestamp": "Timestamp",
+  "monitoring-state": "Status",
+  "monitoring-error": "Error"
```

Comment:
Why put these things in the nl-be translation file but then keep the english words for timestamp and error.
Don't think the monitoring page needs a translation since all the errors we see will be in english anyway.

Reply:
I would keep them here just incase it might someday be used by other people that for example speak french. If we have a translation system, we better use it to its full extend throughout the application. Better have the translation in there then not having it there and have to redo it in the future.